### DISCUSSION TITLE
Matrix group for support

### DISCUSSION THREAD

**Discussion by techrivertree** [mealie-recipes/mealie#3617] [Show and tell]
I created a matrix group for support so I do not have log in the github every time I require assistance.

https://matrix.org/

The group is end to end encrypted. If the devs want to be admins, tell me what your matrix userid is.

Matrix chat link - https://matrix.to/#/#Mealie/self-hosted:matrix.org (I use Element to show chats).

**Comment by fgossel** [comment]
- no, such a room on matrix.org does not exist, and the room address format is incorrect
- with end-to-end encryption, you will have unnecessary issues with a public room

**Reply by techrivertree** [reply]
I do not know why the address for the group does not work. Here is a screenshot of the group.
![Screenshot from 2024-07-24 18-50-59](https://github.com/user-attachments/assets/682b54dc-5253-4acb-980f-57b432d280cb)

I will turn off encryption.

**Comment by techrivertree** [comment]
Encryption cannot be disabled. 
![Screenshot from 2024-07-24 18-52-34](https://github.com/user-attachments/assets/0b318f29-b8d2-4a04-8a3f-f89f003d8249)

**Comment by fgossel** [comment]
Okay, so I found you. The link to the room is [https://matrix.to/#/#mealie-support](https://matrix.to/#/#mealie-support:matrix.org)
[.org](https://matrix.to/#/#mealie-support:matrix.org).

For public encrypted rooms, there are complications with managing user keys and so on and so on. Changing it to non-encrypted is not really possible; doing so might require creating a new room with different settings
